expression/builtin_string.go

```diff
@@ -756,3 +756,33 @@ func builtinCharLength(args []types.Datum, _ context.Context) (d types.Datum, er
 		return d, nil
 	}
 }
+
+// See http://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_find-in-set
+func builtinFindInSet(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	// args[0] -> Str
+	// args[1] -> StrList
+	if args[0].IsNull() || args[1].IsNull() {
+		return
+	}
+
+	str, err := args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	strlst, err := args[1].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	d.SetInt64(0)
+	if strings.Contains(str, ",") {
```

Comment:
Can this line be removed? If str doesn't contain "," , for range strings.Split will do nothing and return ?